docs/pages/jpsreport/jpsreport_inifile.md

```diff
@@ -0,0 +1,373 @@
+| *B* | ![Method B]({{ site.baseurl }}/images/jpsreport_Method_B.png) | $$\langle v \rangle_i$$ and $$\langle \rho \rangle_i$$ |
+| *C* | ![Method C]({{ site.baseurl }}/images/jpsreport_Method_C.png) | $$\langle v \rangle_{\Delta x}$$ and $$\langle \rho \rangle_{\Delta x}$$ |
+| *D* | ![Method D]({{ site.baseurl }}/images/jpsreport_Method_D.png) | $$\langle v \rangle_v$$ and $$\langle \rho \rangle_v$$ |
+| *J* | ![Method J]({{ site.baseurl }}/images/jpsreport_Method_D.png) | $$\langle v \rangle_v$$ and $$\langle \rho \rangle_v$$ |
+
+  Further information relating to each method can be found
+  in [Pedestrian fundamental diagrams: Comparative analysis of experiments in different geometries](http://hdl.handle.net/2128/4898).
+
+`Method_I` can be used for time-series analysis of individual data.
+
+
+### Method A
+For definition see [Method A](jpsreport_method_A.html). Method A is used to analyze the steady state.
+
+```xml
+  <method_A enabled="true">
+    <measurement_area id="2" frame_interval="100"/>
+    <measurement_area id="3" frame_interval="100"/>
+  </method_A>
+```
+
+Possible parameters are:
+- `id` specifies the location of the reference line.
+several measurement areas can be set in one inifile with different id-numbers.
+
+- `frame_interval` specifies the size of time interval (in *frame*)
+   for calculating flow rate.
+
+- up to version 0.8.5: `plot_time_series` specifies whether output the ($$N-t$$)-Diagram.
+
+Possible output data are:
+- `/Fundamental_Diagram/FlowVelocity/`: the output files `Flow_NT_traj_` and `FDFlowVelocity_traj_`.
+
+
+### Method B
+For definition see [Method B](jpsreport_method_B.html). Method B is used to analyze the steady state.
+
+```xml
+  <method_B enabled="false">
+      <measurement_area id="1" />
+  </method_B>
+```
+
+This method can only be used to analyze one directional (or part of one directional) pedestrian movement in corridors. The speed is defined by the length of the measurement area `length_in_movement_direction` and the time a pedestrian stays in the area.
+
+Possible parameters are:
+  - `measurement_area` given by an `id` number.
+     Note that the measurement area for method_B should be
+     rectangle based on the definition of the method.
+
+Possible output data are:
+  - `/Fundamental_Diagram/TinTout/`: output file `FDTinTout_traj_` with mean density and velocity of each pedestrian (PersID, $$\rho_i$$ and $$v_i$$).
+
+
+### Method C
+For definition see [Method C](jpsreport_method_C.html). Method C is used to analyze the steady state.
+
+```xml
+  <method_C enabled="true">
+      <measurement_area id="1"/>
+  </method_C>
+```
+
+Possible parameters are:
+
+- `id` indicates the size and location of the measurement_area.
+   Several measurement areas can be set in one inifile.
+
+- up to version 0.8.5:  `plot_time_series` specifies whether output the ($$\rho-t$$) and ($$v-t$$) diagrams.
```

Comment:
The version number must be corrected.

Reply:
here again, good question. After the union of jpscore and jpsreport, @zeroset and you (@chraibi) should clarify this. JPScore is currently at 0.8.4. JPSreport was already at 0.8.5.

Reply:
see [suggestion](https://github.com/JuPedSim/jpscore/pull/626#discussion_r360882255)